terraform/20-app/ecs.service.front-end.tf

```diff
@@ -30,7 +37,14 @@ module "ecs_service_front_end" {
       essential                              = true
       readonly_root_filesystem               = true
       image                                  = module.ecr_front_end_ecs.image_uri
-      port_mappings                          = [
+      mount_points = [
+        {
+          sourceVolume  = "tmp"
+          containerPath = "/app/.next/cache"
```

Comment:
As part of the recent change that went in which sets the filesystem of all container workloads to be read-only by default, this mounts a writeable space into this location so that next can still use its cache (not in the public dash we don't use this)